src/components/NewQuestionnaire/Questions.vue

```diff
@@ -110,12 +140,41 @@ export default defineComponent({
       questions,
       addQuestion,
       updateQuestions,
+      updateQuestionType,
       deleteQuestion,
       isTextForm,
       isChoiceForm,
       isLinearScaleForm,
-      swapQuestions
+      swapQuestions,
+      copyQuestion
     }
   }
 })
 </script>
+
+<style lang="scss" module>
+.question {
+  display: flex;
+  flex-direction: row;
+  padding: 1rem;
+}
+.border {
+  width: 1px;
+  background-color: $ui-primary;
+  margin-right: 4px;
+}
+.questiondetail {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  gap: 8px;
+}
+.control {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+.button {
+  margin-left: auto;
+}
```

Comment:
```suggestion
.controlButton {
  margin-left: auto;
}
```

あと中央で揃えたいので
```
display: flex;
align-items: center;
```
ほしいです

Reply:
あとgap要りますね